webrtc/README.md

```diff
@@ -11,170 +11,27 @@ Interest Group: [@marten-seemann]
 [@marten-seemann]: https://github.com/marten-seemann
 [@mxinden]: https://github.com/mxinden/
 
-<!-- markdown-toc start - Don't edit this section. Run M-x markdown-toc-refresh-toc -->
-**Table of Contents**
-
-- [WebRTC](#webrtc)
-  - [Motivation](#motivation)
-  - [Addressing](#addressing)
-  - [Connection Establishment](#connection-establishment)
-    - [Browser to public Server](#browser-to-public-server)
-  - [Multiplexing](#multiplexing)
-    - [Ordering](#ordering)
-    - [Head-of-line blocking](#head-of-line-blocking)
-    - [`RTCDataChannel` negotiation](#rtcdatachannel-negotiation)
-    - [`RTCDataChannel` label](#rtcdatachannel-label)
-  - [Connection Security](#connection-security)
-  - [Previous, ongoing and related work](#previous-ongoing-and-related-work)
-  - [Test vectors](#test-vectors)
-    - [Noise prologue](#noise-prologue)
-      - [Both client and server use SHA-256](#both-client-and-server-use-sha-256)
-- [FAQ](#faq)
+WebRTC flavors in libp2p:
 
-<!-- markdown-toc end -->
+1. [WebRTC browser-to-server](./browser-to-server.md)
```

Comment:
Is the name of this spec going to stay as is?

Reply:
That was my intention thus far. I don't have a strong opinion on it. I don't think the name by itself is enough for newcomers to understand the value of the protocol. I am betting on the title plus the short paragraph below.

Reply:
Renamed to `WebRTC Direct`. Resolving here.